endpoint/common.php

```diff
@@ -98,7 +103,8 @@ function find_content($config){
 			),
 			);
 		report_error($details);
-		header('HTTP/1.0 500 Server Config Error',true,500);
+    header('HTTP/1.0 500 Server Config Error',true,500);
+    write_to_kinesis($GLOBALS['kinesis_stream_name'], '{"access_url":"'.$GLOBALS['actual_link'].'", "output_message":null, "response_code":500, "php_headers":'.json_encode(headers_list()).'}', null);
```

Comment:
surely php has nicer ways to serialize json than this?! Might as well leave it if it's working, I suppose, but there is always a risk that a dodgy character will slip through and break any reader

Reply:
Are you refering to my use of the json_encode() function or the use of strings to generate JSON?

Reply:
strings to generate json! surely you could set up the whole thing as some kind of dictionary/object and use `json_encode` on that?

Reply:
That is possible yes. I am not sure how it would help.

Reply:
Don't worry about it in this specific instance, but I would always do that in preference to manually coding strings. The reason is that then you should be guaranteed to be able to _parse_ it on the other side - you won't accidentally end up with unescaped characters breaking the parser.